app/views/membership_applications/_reason_waiting.html.haml

```diff
@@ -38,7 +36,7 @@
 
 
   function selected_is_customOtherReason() {
-    return reasons_list.value === other_reason_option.value;
+    return $('#member_app_waiting_reasons option:selected').text() === other_reason_option.text;
```

Comment:
Your statement did not work for me - it could be because I converted to query(?).